javascript-conditionals/main.js

```diff
@@ -0,0 +1,81 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro,
+            recommendMovie
+ */
+
+const isUnderFive = (num) => {
+  return (num < 5);
+};
+
+const isEven = (num) => {
+  return (num % 2 === 0);
+};
+
+const startsWithJ = (str) => {
+  return (str[0] === 'J');
+};
+
+const isOldEnoughToDrink = (person) => {
+  return (person.age > 20);
```

Comment:
What's the legal drinking age 👮🏻 ? We're talking about **legal** age, not when you started drinking 😉